21-sump-level-monitor/README.md

```diff
@@ -0,0 +1,103 @@
+# Sump Level Monitor
+
+Monitor water level changes in a sump using a Notecard and a float switch.
+
+- [Sump Level Monitor](#sump-level-monitor)
+  - [You Will Need](#you-will-need)
+  - [Notehub Setup](#notehub-setup)
+  - [Hardware Setup](#hardware-setup)
+  - [Notecard Firmware Setup](#notecard-firmware-setup)
+  - [Notecard Configuration](#notecard-configuration)
+    - [Notecard CLI](#notecard-cli)
+  - [Testing](#testing)
+  - [Going Further](#going-further)
+
+## You Will Need
+
+* USB A to micro USB cable
+* [Notecarrier-A](https://shop.blues.io/products/carr-al)
+* [Notecard](https://blues.io/products/notecard/)
+* [Sump Alarm Three Wire Float Switch](https://www.sumpalarm.com/products/float-switch-for-use-septic-system-sump-pump-water-tank-see-all-five-cable-length-options)
+* Breadboard
+* Soldering iron and solder flux
+* Strip of male header pins
+
+## Notehub Setup
+
+Sign up for a free account on [notehub.io](https://notehub.io) and [create a new project](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-pi/#set-up-notehub).
+
+## Hardware Setup
+
+1. Assemble Notecard and Notecarrier as described [here](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-f/).
+2. Strip ~1/2" of the insulation off the float switch's brown and black wires. Twist the strands of each wire tightly and then use a soldering iron to [tin each wire](https://www.youtube.com/watch?v=pRPF4wpXX9Q). We will not be using the remaining blue wire.
+3. With the float switch wires tinned, break off two male header pins and solder a header onto each switch wire so that you can easily connect the switch to a breadboard.
+4. Connect a jumper from GND on the Notecarrier to one row of the breadboard. Connect one of the float switch's wires (it doesn't matter which one) to this same row.
+5. Connect another jumper from AUX1 on the Notecarrier to a separate row on the breadboard. Connect the remaining float switch wire to this same row. With this configuration, when the switch is closed, AUX1 will be connected go ground.
```

Comment:
It would be nice with a frizing diagram or similar to show the circuit. Is the breadboard strictly necessary? If I'm not mistaken it seems the float switch could be connected directly to GND and AUX1?

Reply:
You're right. Should be able to just plug the soldered-on headers directly into the Notecarrier. That way's simple enough that I think a diagram is probably unnecessary. Will change.